README.md

```diff
@@ -45,6 +45,8 @@ NAME        | Description
 ------------|-------------
 OUTPUT_DIR  | Sets the location of the output directory
 NPM_BUILD_COMMAND | Override the default "npm run build"
+NPM_RUN_COMMAND | Override the default "npx serve" command
```

Comment:
good call,  then should that NPM_BUILD_COMMAND also change to NPM_BUILD?  at this point i'm not to worried about breaking previous versions since we are in active dev

Reply:
> should that NPM_BUILD_COMMAND also change to NPM_BUILD

Yeah, probably so.